src/Layout.tsx

```diff
@@ -32,7 +44,12 @@ const Layout = () => {
     <main className="h-screen flex pt-6 md:pt-0 md:items-center justify-center font-sans">
       <Outlet context={{ channelName: channelId, clientId, setProjectInfo }} />
       <div className="fixed bottom-0 md:absolute md:left-12 md:bottom-12">
-        <InfoCard projectInfo={projectInfo} />
+        <InfoCard
+          projectInfo={projectInfo}
+          openNewTabTrackerEvent={() => openNewWindow(projectInfo.name)}
+          githubRepoLinkTrackerEvent={() => viewRepoAtGithub(projectInfo.name)}
+          allExamplesLinkTrackerEvent={() => viewAllExamples(projectInfo.name)}
```

Comment:
Maybe the names of these props should be closer to the names of the functions (`openNewTabTrackerEvent` -> `openNewWindow`, why not `openNewTabTrackerEvent` -> `openNewTabTrackerEvent`)? They seem to be saying the same thing, but in different ways.

Reply:
that makes sense. @dpiatek 
I was also considering having just one function & passing the event to it. Or would you say it's better to leave it explicit as different functions like this?